src/Counter/Counter.util.test.ts

```diff
@@ -1,7 +1,15 @@
-// import {formatNumberToString} from './Counter.util'
+import {formatNumberToString} from './Counter.util'
 
 describe('Unit::Counter.util', () => {
     describe('formatNumberToString', () => {
-     
+        it('should return a string', () => {
+            const result1 = formatNumberToString(0)
+            const result2 = formatNumberToString(1)
```

Comment:
Chaque it doit testé seulement un seul cas / contexte. 

Là dans ce test tu couvres 3 cas donc tu devrais avoir 3 its.

Pourquoi ? dans un premier temps ça permet de séparer le test histoire qu'il ne soit pas trop conséquent. dans un second temps pour la lisibilité / documentation.